mail_activity_partner/views/mail_activity_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+<!--
+    VIEWS
+-->
+
+    <!-- FORM VIEW -->
+    <record id="mail_activity_view_form_board" model="ir.ui.view">
+        <field name="name">mail.activity.boards.view.form</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail_activity_board.mail_activity_view_form_board"/>
+        <field name="arch" type="xml">
+            <field name="activity_type_id" position="before">
+                <field name="partner_id" readonly="True"/>
+                <field name="commercial_partner_id"/>
+            </field>
+        </field>
+    </record>
+
+
+    <!-- TREE VIEW -->
+    <record id="mail_activity_view_tree" model="ir.ui.view">
+        <field name="name">mail.activity.boards.view.tree</field>
+        <field name="model">mail.activity</field>
+        <field name="inherit_id" ref="mail_activity_board.mail_activity_view_tree"/>
+        <field name="arch" type="xml">
+            <field name="activity_type_id" position="before">
+                <field name="partner_id" readonly="True"/>
+                <field name="commercial_partner_id"/>
+            </field>
+        </field>
+    </record>
+
+
+    <!-- KANBAN VIEW -->
+     <record id="mail_activity_view_kanban" model="ir.ui.view">
+         <field name="name">mail.activity.boards.view.kanban</field>
+         <field name="model">mail.activity</field>
+         <field name="inherit_id" ref="mail_activity_board.mail_activity_view_kanban"/>
+         <field name="arch" type="xml">
+             <strong class="o_kanban_record_title" position="inside">
+                 <field name="partner_id"/>
+                 <field name="commercial_partner_id"/>
```

Comment:
I would remove 'commercial_partner_id' from the kanban view. It is not necessary. If the partner is a contact of the company, it will be displayed already as part of the partner_id display name.

![image](https://user-images.githubusercontent.com/7683926/50589593-971aca80-0e87-11e9-819c-31c80d559d63.png)